src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Unit.Tests/CompilationHelper.cs

```diff
@@ -22,27 +23,30 @@ public class CompilationHelper
             // workaround https://github.com/dotnet/roslyn/pull/55866. We can remove "LangVersion=Preview" when we get a Roslyn build with that change.
             .WithLanguageVersion(LanguageVersion.Preview);
 
+#if NETCOREAPP
+        private static readonly Assembly systemRuntimeAssembly = System.Runtime.Loader.AssemblyLoadContext.Default.LoadFromAssemblyName(new AssemblyName("System.Runtime"));
```

Comment:
```suggestion
        private static readonly Assembly systemRuntimeAssembly = Assembly.Load(new AssemblyName("System.Runtime"));
```
This does the same thing.